action/signedaction.go

```diff
@@ -19,17 +19,29 @@ var (
 	ValidSig, _ = hex.DecodeString("15e73ad521ec9e06600c59e49b127c9dee114ad64fb2fcbe5e0d9f4c8d2b766e73d708cca1dc050dd27b20f2ee607f30428bf035f45d4da8ec2fb04a90c2c30901")
 )
 
+type SignedActionOption func(*EnvelopeBuilder)
+
+func WithChainID(chainID uint32) SignedActionOption {
+	return func(b *EnvelopeBuilder) {
+		b.SetChainID(chainID)
+	}
+}
```

Comment:
IMO this is not a very needed or helpful change.
real use of `SetChainID()` is only 1 place at ioctl SendAction

Reply:
will be used for e2etest